### PR TITLE
BAU: improve card capture process logging

### PR DESCRIPTION
**Pull request by jankowiakmaria**
* added chargeID to MDC for capture process

### CODE REVIEW

**Review by kbottla — APPROVED** [review]
LGTM!

File: src/main/java/uk/gov/pay/connector/paymentprocessor/service/CardCaptureProcess.java [hunk 1]
```diff
@@ -36,19 +37,20 @@ public void handleCaptureMessages() throws QueueException {
         List<ChargeCaptureMessage> captureMessages = captureQueue.retrieveChargesForCapture();
         for (ChargeCaptureMessage message : captureMessages) {
             try {
-                LOGGER.info("Charge capture message received - [externalChargeId={}] [queueMessageId={}] [queueMessageReceiptHandle={}]",
-                        message.getChargeId(),
+                MDC.put("chargeId", message.getChargeId());
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/connector/paymentprocessor/service/CardCaptureProcess.java:40]: As part of the structured logging work, we’re trying to define a consistent set of keys for things like this:

https://github.com/alphagov/pay-java-commons/blob/master/logging/src/main/java/uk/gov/pay/logging/LoggingKeys.java

For this, adding `"external_id"` and also `"resource_type"` with a value of `"payment"` and `"gateway_card_operation"` with a value of `"Capture"` would fit in with the pattern.

(The keys are very new and aren’t being used much yet, so if it requires too much contortion to use the pre-defined keys, we probably want to refine the pre-defined keys.)

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/connector/paymentprocessor/service/CardCaptureProcess.java:40]: I forgot: more details about the approach taken with the keys can be found in commit https://github.com/alphagov/pay-java-commons/commit/37ca42954cfa19a0649a9df7e0b3b07695ae5fa9.

Comment (jankowiakmaria) [reply src/main/java/uk/gov/pay/connector/paymentprocessor/service/CardCaptureProcess.java:40]: I might add it separately (unless someone pick up this work first). This is just a quick side PR (while waiting for PR review).
Thanks for the heads up though.

**Pull request merged**